verifiers/src/yang/verifiers/example_verifier.py

```diff
@@ -0,0 +1,66 @@
+from typing import List
```

Comment:
This is a valid verifier so don't name it "example".  How about count_verifier.py?